### ISSUE TITLE
Please rename this lib: Routex already in use

### ISSUE THREAD

**Issue by BartOtten** [handlecommerce/routex#1]
Dear maintainer(s),

We share our love for routes in Elixir, and our love for Routex! Unfortunately, we both have another Routex in mind. I am talking about https://hex.pm/packages/routex

Could you be so kind to rename this lib/repo?

Regards,
Bart Otten, author of 'different Routex'

**Comment by markglenn** [comment]
I understand your concern, but this project was created before the other Routex. It's also an internal project open to others if needed.

**Comment by BartOtten** [comment]
Hi Mark,  I get reports the name conflict is causing issues with AI mixing my libs and this repo even when explicitly told to ignore this repository.

Although the repo itself might be newer, the name was different. As a result I did not find it when I named my lib and published at hex (the readme says “not -yet- at hex” btw.

https://github.com/handlecommerce/routex/commit/4144f2418b8e42c4fa78b0a0a551b539afdec0fd

I understand it may cause inconvenience, but as it is used internal I guess it is easier than me renaming a public library, creating a new logo and new topics at Elixir forum. Maybe renaming the public repository is enough..?

Hope you understand,
Bart
